localstack/services/awslambda/invocation/runtime_executor.py

```diff
@@ -108,17 +112,23 @@ def __init__(self, message: str):
         super().__init__(message)
 
 
-class RuntimeExecutor:
-    id: str
-    function_version: FunctionVersion
+class DockerRuntimeExecutorPlugin(RuntimeExecutorPlugin):
+    name = "docker"
+
+    def load(self, *args, **kwargs):
+        return DockerRuntimeExecutor
```

Comment:
remember that, if you put a plugin in a module, the entire module will be imported when the plugin resolves, which is potentially when localstack starts up. this is why we've made it a common pattern to move plugins into their own modules, and defer any necessary imports into the `load` method - i would suggest the same here. :-)

Reply:
Moved it to its own "plugins.py" file! Thanks for the suggestion!